src/components/taskpage/TaskPage.jsx

```diff
@@ -52,16 +43,14 @@ export default function TaskPage() {
                             <TaskItem
                                 key={task.id}
                                 {...task}
-                                toggle={() => setIsEditTask((s) => !s)}
-                                setTaskValues={setTaskValues}
                             />
                         );
                     })}
                 </div>
             </div>
             {isModal ? <AddTask toggleModal={toggleModal} /> : ""}
-            {isEditTask ? (
-                <EditTask toggleModal={()=>setIsEditTask(s=>!s)} {...taskObject} />
+            {editData.id ? (
+                <EditTask  />
```

Comment:
here you can use "&&" operator as you are not setting anything on the other side of the ternary

Reply:
done